src/lib/Rattletrap/Type/ObjectTarget.hs

```diff
@@ -0,0 +1,53 @@
+module Rattletrap.Type.ObjectTarget where
+
+import qualified Rattletrap.BitGet as BitGet
+import qualified Rattletrap.BitPut as BitPut
+import qualified Rattletrap.Schema as Schema
+import qualified Rattletrap.Type.I32 as I32
+import qualified Rattletrap.Utility.Json as Json
+
+data ObjectTarget = ObjectTarget
```

Comment:
This type could be used in a few more places, but I don't want to change the schema.